stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -52,6 +55,9 @@ import io.getstream.chat.android.client.models.User
  * - Customizing the click and longCLick (via the adapter)
  * - The list_item_message template to use (perhaps, multiple ones...?)
  */
+
+private val LOADING_VIEW_ID = R.id.message_list_loading_view_id
```

Comment:
This clearly belongs to the class itself (as opposed to the file), I would prefer we place it within the class. Makes the class level comment's position weird as well.